packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
+      {{! Hds-Tab }}
+      <li class="hds-tab" role="presentation">
+        <a
+          role="tab"
+          href="#section1"
+          id="tab1"
+          {{! aria-selected="true"  TODO: Set by arg: "isSelected" }}
+          {{! tabindex="-1"  TODO: Set if not aria-selected ("isSelected") }}
+        >
+          Tab 1
+          {{this.isSelected}}
+        </a>
+      </li>
+      <li class="hds-tab" role="presentation">
+        <a
```

Comment:
we're eliminating the use of tabs for navigation, so the markup that @KristinLBradley has so far is correct from that perspective. 👍

Reply:
Thanks @MelSumner

Reply:
Thanks @didoo, I'm going to keep the structure as is for now. I don't think this level of navigation necessarily requires a nav element (it's mainly used for primary page navigation) and I don't want to misuse li's to trigger behavior instead of using semantic elements such as an anchor tag or button.